pkg/apiserver/rest/apis/v1alpha2/catalog.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package apis
+package v1alpha2
```

Comment:
why move to v1alpha2?  I think the api should start from v1.

Reply:
The API path starts with `/v1`.
The data model version follows the OAM version.